### ISSUE TITLE
does not seem to work at all, not sure what is going on

### ISSUE THREAD

**Issue by corsen2000** [atom/spell-check#57]
Given:

```markdown
this is not spelllzzed right
```

There is no highlight.  If I right click the word and select "Correct Spelling" nothing happens.

The package is enabled and I have not modified the "grammars" setting.

I am using Atom 0.194.0 and have launched in both safe mode and normal.

Not sure what is going on.  Any suggestions?

Thank you.

**Comment by mnquintana** [comment]
What file type were you trying to use it with? By default it currently only supports the following grammars (from the Readme):
> - Plain Text
> - GitHub Markdown
> - Git Commit Message

**Comment by corsen2000** [comment]
My rile is `README.md` and it is being detected as GitHub Markdown.

![image](https://cloud.githubusercontent.com/assets/733656/7421531/3b8997d4-ef53-11e4-974c-c67721be26aa.png)

**Comment by lee-dohm** [comment]
What OS and version are you running?

**Comment by corsen2000** [comment]
* Windows 7
* Atom 0.194.0
* spell-check 0.55.0

**Comment by izuzak** [comment]
Works for me on Atom 0.197.0-f8047ba on Windows 8.1.

![screen shot 2015-05-01 at 21 37 43](https://cloud.githubusercontent.com/assets/38924/7436114/683b884e-f04a-11e4-8405-8f23f341b497.png)

**Comment by SavageReborn** [comment]
Just stated using atom, and this is an issue for me as well ![alt text](http://puu.sh/hAWKd/8ddc71e84c.png "doesn't work").

Windows 7, Atom 0.196.0, spell-check 0.56.0

Tried adding all of the grammars manually instead of leaving it as default, still doesn't work.

Surpisingly enough pulling up the command pallete does not show spell check. Not sure if it was some sort of installation problem, I tried disabling and reenabling it and it still nothing.

**Comment by izuzak** [comment]
Interesting, it seems to work just fine on Windows 8.1 but I don't have Windows 7 to confirm it doesn't work there. 

@SavageReborn @corsen2000 Did spell-check work for you before and it stopped working recently, or did it never work?

**Comment by SavageReborn** [comment]
@izuzak Never worked before. Just installed atom recently.

**Comment by corsen2000** [comment]
It definitely used to work for me.  I'm not sure exactly when it stopped, probably with the last ~4 weeks or so.

**Comment by thomasjo** [comment]
`spellchecker` (library used by this package) behaves differently for Windows 7 and 8.x (and above). For Windows 7 and below, as well as Linux, the library relies on `hunspell`. There might be some issue related to that. Note that `hunspell` is bundled with `spellchecker`.

**Comment by izuzak** [comment]
@thomasjo Yep, my intuition was nudging me in that direction as well -- seems like something broke recently. 

@kevinsawicki @paulcbetts Do you maybe have a Windows 7 machine you could try this out on, or maybe know what might be causing this?

**Comment by paulcbetts** [comment]
@izuzak Set the SPELLCHECKER_PREFER_HUNSPELL env var to 1 to get the Win7 behavior on Win8

**Comment by paulcbetts** [comment]
Also maybe this is the culprit? https://github.com/atom/node-spellchecker/blob/master/src/spellchecker_hunspell.cc#L32

**Comment by izuzak** [comment]
Yep, I see it now -- thanks @paulcbetts! :sparkler:  

Seeing these in console now:

```
[3136:0505/101201:INFO:CONSOLE(114)] "Error starting spell check task", source: C:\Program Files\Atom\resources\app.asar\node_modules\spell-check\lib\spell-check-view.js (114)
```

**Comment by paulcbetts** [comment]
Also, my psychic debugger is saying that you're trying to hand Hunspell a dictionary path that's in an ASAR package, which will definitely not work

**Comment by jonathandelgado** [comment]
+1

**Comment by X-G** [comment]
windows 10 does not work.

**Comment by Ehekatl** [comment]
windows 10 does not work + 1

**Comment by izuzak** [comment]
@X-G @Ehekatl Can you describe what _exactly_ you're doing to reproduce the problem? Which type of file are you opening? Can you provide a screenshot which shows where you expected to see the spellcheker to kick in and also the type of the file that's opened (bottom right corner)?

**Comment by jonathandelgado** [comment]
Windows 10 is working for me - do you have any errors in console @Ehekatl?

**Comment by X-G** [comment]
@Ehekatl spell check works on my Windows 10 laptop right now.

**Comment by Ehekatl** [comment]
@X-G what did you do?

@izuzak @jonathandelgado I'm using default setting in spell-check package, the spell-check didn't work for all kinds of file, even I define the file scopes in setting and reinstall reactive it again and again it's still not work, with the same atom version same setting and same file, it work on my linux machine, I can't see any errors in console, how could I debug this

**Comment by X-G** [comment]
@Ehekatl I did nothing, it just works. I think the reason should be I upgrade windows 10 or atom.

**Comment by Ehekatl** [comment]
I just reinstall a brand new windows 10 on my laptop, problem still, no errors in console, any help here? It that possibly relate my OS language(it's chinese) ?

**Comment by cormeijneke** [comment]
Same here. Windows 10, Atom 10.0.19, spell check not working.

**Comment by paulcbetts** [comment]
@Ehekatl Yes, if you don't have an English language dictionary on your OS for >= Win8, you'll have to install it before spellcheck will work

**Comment by cormeijneke** [comment]
OK, Thanks!

Installing the US English language pack did the trick. I followed the steps as explained in the following link: http://www.tenforums.com/tutorials/3813-language-add-remove-change-windows-10-a.html

After restarting Atom the spell check works.

**Comment by Ehekatl** [comment]
@paulcbetts Thx, it works! : )

**Comment by abinashmohanty** [comment]
This is how one can fix the spell check issue. Go to packages > select spell-check > click settings and add scopes per language. Like here in this example, I have added for html and css and that would be _text.html.basic, source.css_. You'll see the changes once it's done. 

<img width="785" alt="atom spell check" src="https://cloud.githubusercontent.com/assets/8434934/19556220/bb240bc2-96dd-11e6-8bfe-549a2b81ba91.png">

**Comment by ThomDietrich** [comment]
To extend on @abinashmohanty 's answer: If in doubt about the scope to add to that list:

Go back to the document whose type you want to add, open the command palette, then search and execute "Editor: Log Cursor Scope"

![grafik](https://user-images.githubusercontent.com/2870104/30585891-81c47986-9d2e-11e7-9f47-db49003bda1e.png)

A notification will show the scope(s) at your current cursor position.

**Comment by Tao54** [comment]
@abinashmohanty thanks so much for the help!
